lib/omniauth/strategies/azure_activedirectory_v2.rb

```diff
@@ -14,41 +16,45 @@ class AzureActivedirectoryV2 < OmniAuth::Strategies::OAuth2
       args [:tenant_provider]
 
       def client
-        if options.tenant_provider
-          provider = options.tenant_provider.new(self)
-        else
-          provider = options  # if pass has to config, get mapped right on to options
-        end
+        provider = if options.tenant_provider
+                     options.tenant_provider.new(self)
+                   else
+                     options # if pass has to config, get mapped right on to options
+                   end
```

Comment:
I'm not against seeing formatting changes in PRs where those reformats aren't part of the core fix. I do it myself often enough so it would be hypocritical to object! The style of much of the code in this gem was inherited from https://github.com/marknadig/omniauth-azure-oauth2/blob/master/lib/omniauth/strategies/azure_oauth2.rb and there's nothing wrong with some tidying.

That said, if we're going to do that, I'd prefer to see a slightly different code style on a couple of points:

* Indenting the 'if' conditions as above is fragile if the indent is not then at a precise multiple of the indent width; in this case, we indent only with 2 spaces so that's quite easy to match, but if you're left trying to line things up at an odd column number you can end up fighting your text editor. And that _is_ the case here - we end up aligned on column 19 (counting from zero).
* If changes from e.g. `foo if bar` to multiline `if bar; foo; end` are being made, then they _must_ be applied consistently.

On the first point I do prefer your `provider = if...` approach, but I'd just like to see it all indented to align with `provider`:

```ruby
      def client
        provider = if options.tenant_provider
          options.tenant_provider.new(self)
        else
          options # if pass has to config, get mapped right on to options
        end
```

Reply:
(Later comment: That 'if pass has to config' comment from https://github.com/marknadig/omniauth-azure-oauth2/blob/master/lib/omniauth/strategies/azure_oauth2.rb doesn't make sense; I wonder if it means "if passing a Hash to config, map it directly to options" or something?! 🤷 )

Reply:
Maybe the comma is like a semi-colon? 

if passed, we have to config; else map it right onto options

Reply:
I removed it since it sounds kinda redundant? But happy to add a better comment to replace it if you have any ideas